lib/rollbar/item/locals.rb

```diff
@@ -0,0 +1,49 @@
+require 'rollbar/notifier'
+
+module Rollbar
+  class Item
+    class Locals # :nodoc:
+      class << self
+        def exception_frames
+          Rollbar.notifier.exception_bindings
+        end
+
+        def locals_for_location(filename, lineno)
+          puts "Location: #{filename}:#{lineno}"
+          if (frame = frame_for_location(filename, lineno))
+            puts "Frame: #{frame[:path]}:#{frame[:lineno]}"
+            locals_for(frame[:binding]).tap { |locals| puts locals.inspect }
+          else
+            {}
+          end
+        end
+
+        def frame_for_location(filename, lineno)
```

Comment:
Consider hiding some of those methods behind `protected` or `private` since they are only used internally. What do you think?

Reply:
Great idea. I'll do that.